File: Python Word Examples/ConvertApiDemo/convertInRequest.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.convert_api.ConvertApi(configuration)

file_name = "D:/input/convert.docx"
target_format = 'postScript' #Available values: doc, docx, wordml, wordxml, odt, docm, dotm, dotx, dot, rtf, pdf, xps, epub, pcl, postScript
extra = {'password': ''}
result = api.convert_in_request(file_name, target_format, **extra)

File: JavaScript Word Examples/ConvertApiDemo/convertInRequest.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword= require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.ConvertApi(apiClient);

    var fs = require('fs');
    var file = fs.createReadStream("D:/Convert.docx");
    var format = "pdf"; //Available values : doc, docx, wordml, wordxml, odt, docm, dotm, dotx, dot, rtf, pdf, xps, html, epub, pcl, postScript
    var opt = {};
    instance.convertInRequest(file, format, opt,
        function (error, data, response) {
            if (error) throw error;
        });
})();